hbase-it/src/test/java/org/apache/hadoop/hbase/rsgroup/IntegrationTestRSGroup.java

```diff
@@ -1,99 +0,0 @@
-/**
- * Copyright The Apache Software Foundation
- *
- * Licensed to the Apache Software Foundation (ASF) under one
- * or more contributor license agreements.  See the NOTICE file
- * distributed with this work for additional information
- * regarding copyright ownership.  The ASF licenses this file
- * to you under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance
- * with the License.  You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package org.apache.hadoop.hbase.rsgroup;
-
-import org.apache.hadoop.hbase.HConstants;
-import org.apache.hadoop.hbase.IntegrationTestingUtility;
-import org.apache.hadoop.hbase.Waiter;
-import org.apache.hadoop.hbase.testclassification.IntegrationTests;
-import org.junit.After;
-import org.junit.Before;
-import org.junit.experimental.categories.Category;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-/**
- * Runs all of the units tests defined in TestGroupBase as an integration test.
- * Requires TestRSGroupBase.NUM_SLAVE_BASE servers to run.
- */
-@Category(IntegrationTests.class)
-public class IntegrationTestRSGroup extends TestRSGroupsBase {
-  private final static Logger LOG = LoggerFactory.getLogger(IntegrationTestRSGroup.class);
-  private static boolean initialized = false;
-
-  @Before
-  public void beforeMethod() throws Exception {
-    if (!initialized) {
-      LOG.info("Setting up IntegrationTestRSGroup");
-      LOG.info("Initializing cluster with " + NUM_SLAVES_BASE + " servers");
-      TEST_UTIL = new IntegrationTestingUtility();
-      TEST_UTIL.getConfiguration().set(HConstants.HBASE_MASTER_LOADBALANCER_CLASS,
-        RSGroupBasedLoadBalancer.class.getName());
-      RSGroupUtil.enableRSGroup(TEST_UTIL.getConfiguration());
-      ((IntegrationTestingUtility) TEST_UTIL).initializeCluster(NUM_SLAVES_BASE);
-      // set shared configs
-      ADMIN = TEST_UTIL.getAdmin();
-      CLUSTER = TEST_UTIL.getHBaseClusterInterface();
-      LOG.info("Done initializing cluster");
-      initialized = true;
-      // cluster may not be clean
-      // cleanup when initializing
-      afterMethod();
-    }
-  }
-
```

Comment:
I believe we should add a `@Test` method (maybe empty for now) to run `@Before` and `@After` because no test is present for quite some time and hence before and after is also not getting executed. 
Anyways, @Apache9 will have better suggestions.

Reply:
yeah, it was used for running many integration tests which now have been split well and this became useless.